src/tray.ts

```diff
@@ -45,23 +45,32 @@ function toggleWin(): void {
     }
 }
 
-interface IConfig {
-    icon_path: string; // eslint-disable-line camelcase
+export interface IConfig {
+    allowWebIconOverride: boolean;
```

Comment:
this doesn't really explain who is being restricted here - can we get a comment?

Reply:
Added!